R/so6.R

```diff
@@ -142,7 +142,7 @@ export_event_so6 <- function(wef, til) {
 
 generate_so6 <- function(event_trajectory) {
   event_trajectory %>%
-    dplyr::mutate(adep = "ZZZZ", ades = "ZZZZ") %>%
```

Comment:
In fact this line is needed when generating trajectories from minimalistic position reports, i.e. those that do not know ADEP and/or ADES like ADS-B reports